src/montecarlo.cpp

```diff
@@ -198,14 +205,13 @@ double IdealTerm(Space &spc_n, Space &spc_o, const Change &change) {
                 }
                 int dN = N_n - N_o;
                 if (dN != 0) {
-                    double V_n = spc_n.geo.getVolume();
-                    double V_o = spc_o.geo.getVolume();
+                    double V = spc_n.geo.getVolume() * 1.0_molar;
                     if (dN > 0)
                         for (int n = 0; n < dN; n++)
-                            NoverO += std::log((N_o + 1 + n) / (V_n * 1.0_molar));
+                            NoverO += std::log( (N_o + 1 + n) / V );
```

Comment:
was the old code wrong? it would seem this could break tests(?)

Reply:
Yes, it should be the volume of the final state but it doesn't matter because RCMC moves don't change the volume. The conversion factor 1.0_molar has moved to line 208.